### PR TITLE
fix: Specify HTML lang tag "en" 📓

### PR DESCRIPTION
**Pull request by darcywong00**
Addresses this point of #383 

> Fix errors regarding the language markup of your webpage
The language is not specified in the HTML markup

We're not yet ready to localize keyman.com, so for now setting the HTMl lang tag to "en".

### CODE REVIEW

File: _includes/2020/templates/Head.php [hunk 1]
```diff
@@ -13,6 +13,9 @@ static function render($fields = []) {
       if(!isset($fields->title)) {
         $fields->title = 'Keyman | Type to the world in your language';
       }
+      if(!isset($fields->language)) {
```
Comment (darcywong00) [inline _includes/2020/templates/Head.php:16]: I think this gives us flexibility for a page to specify `$language='my';` in a header

Comment (darcywong00) [reply _includes/2020/templates/Head.php:16]: From other PRs, checking `empty()` better

```suggestion
      if(!empty($fields->language)) {
```

File: _includes/2020/templates/Head.php [hunk 2]
```diff
@@ -13,6 +13,9 @@ static function render($fields = []) {
       if(!isset($fields->title)) {
         $fields->title = 'Keyman | Type to the world in your language';
       }
+      if(!empty($fields->language)) {
```
Comment (darcywong00) [inline _includes/2020/templates/Head.php:16]: oops - fixing last suggestion
```suggestion
      if(empty($fields->language()) {
```

**Review by mcdurdin — APPROVED** [review]

File: _includes/2020/templates/Head.php [hunk 3]
```diff
@@ -13,6 +13,9 @@ static function render($fields = []) {
       if(!isset($fields->title)) {
         $fields->title = 'Keyman | Type to the world in your language';
       }
+      if(empty($fields->language()) {
```
Comment (mcdurdin) [inline _includes/2020/templates/Head.php:16]:
```suggestion
      if(empty($fields->language) {
```

File: _includes/2020/templates/Head.php [hunk 4]
```diff
@@ -23,7 +26,7 @@ static function render($fields = []) {
         $fields->js = [];
       }
 ?><!DOCTYPE html>
-<html>
+<html lang="<?= $fields->language; ?>">
```
Comment (mcdurdin) [inline _includes/2020/templates/Head.php:29]:
```suggestion
<html lang="<?= $fields->language ?>">
```

File: _includes/2020/templates/Head.php [hunk 5]
```diff
@@ -13,6 +13,9 @@ static function render($fields = []) {
       if(!isset($fields->title)) {
         $fields->title = 'Keyman | Type to the world in your language';
       }
+      if(empty($fields->language) {
```
Comment (mcdurdin) [inline _includes/2020/templates/Head.php:16]:
```suggestion
      if(empty($fields->language)) {
```

**Pull request merged**